lib/index.js

```diff
@@ -22,6 +22,12 @@ var DEFAULT_DELEGATE = {
     fs.mkdirSync(outputPath);
   },
   change: function(inputPath, outputPath, relativePath) {
+    // We no-op if the platform can symlink, because we assume the output path
+    // is already linked via a prior create operation.
+    if (util.canSymlink) {
+      return;
+    }
+
     fs.unlinkSync(outputPath);
```

Comment:
just curious, what happens if the symlink itself merely changes, like for example a new source.

cc @hjdivad